File: Objective-C Version/tvOSCardAnimationDemo/tvOSCardAnimation/ViewController.m
//
//  ViewController.m
//  tvOSCardAnimation
//
//  Created by yangqitao on 15/9/21.
//  Copyright © 2015年 yangqitao. All rights reserved.
//

#import "ViewController.h"
#import "tvOSCardView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    tvOSCardView *cardView = [[tvOSCardView alloc]init];
    cardView.center = self.view.center;
    cardView.bounds = CGRectMake(0, 0, 150, 200);
    [self.view addSubview:cardView];
    
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: Swift Version/tvOSCardAnimation-Swift/tvOSCardAnimation-Swift/ViewController.swift
//
//  ViewController.swift
//  tvOSCardAnimation-Swift
//
//  Created by Kitten Yang on 1/15/16.
//  Copyright © 2016 Kitten Yang. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        let cardView = tvOSCardView()
        cardView.center = view.center
        cardView.bounds = CGRect(x: 0, y: 0, width: 150, height: 200)
        view.addSubview(cardView)

    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }


}

